lib/vanagon/platform/dsl.rb

```diff
@@ -128,6 +128,13 @@ def cross_compiled(cxx_flag)
         @platform.cross_compiled = cxx_flag
       end
 
+      # Set the use_os_toolchain flag for the platform
+      #
+      # @param flag [Boolean] True if this platform uses host OS provided toolchain
+      def use_os_toolchain(flag)
+        @platform.use_os_toolchain = flag
```

Comment:
Re: parsing values. You can do it here, or you can do it when it's called by `platform.rb` but I'd only mangle/parse/match it in one place. You choice.